components/brave_webtorrent/extension/styles/styles.ts

```diff
@@ -9,3 +9,8 @@ export const StyledTorrentViewer = styled('div')<{}>`
   padding: 100px;
   font-family: Muli, sans-serif;
 `
+
+export const StyledButtonContainer = styled('div')<{}>`
```

Comment:
I think a more minimal change would be to add a ButtonsContainer element, with `flex-direction: row;` and put both buttons inside it:

```ts
export const ButtonsContainer = styled.div`
    display: flex;
    flex-direction: row;
`
```

Reply:
You can then use `gap: 10px` for spacing.

Reply:
Sure, I'll change it

Reply:
Thanks! Hopefully this won't be necessary at all with the `:global { ... }` fix!